Cargo.lock

```diff
@@ -44,7 +44,7 @@ checksum = "648ed8c8d2ce5409ccd57453d9d1b214b342a0d69376a6feda1fd6cae3299308"
 dependencies = [
  "proc-macro2",
  "quote",
- "syn",
+ "syn 1.0.109",
```

Comment:
Do the changes really require a lot upgrade of dependencies or it happened accidentally?

If it - it is OK.
If it accidentally - will be better to revert unneed updates - for continue to test with as older libraries as possible (because upgrade at client code much easer then downgrade if it needs).

Reply:
this change is happening automatic after build. Maybe on of the dependencies required it